### PR TITLE
Preserving xmlns attribute in all svg icons file #6107

### PR DESCRIPTION
**Pull request by vicky1312**
- Updates all svg icons with the xmlns attribute
- Updates a code in wagtail/wagtail/admin/homes.py(L300) that removes the xmlns attribute when the files are processed in python.
- This is a partial fix to issue #6107 
- The reason for applying this attribute is  https://github.com/wagtail/wagtail/issues/6107#issuecomment-1276245262

Annotation 2022-10-16 123032
Annotation 2022-10-16 123135

### CODE REVIEW

**Comment by lb-** [comment]
@vicky1312 when raising a PR it's really helpful if you can add a bit of context you.he PR description. 

Thank you so much for prepya PR - just a few tips for helping this get looked at sooner. 

For example
------


- Updates all svg icon files with blah blah thing
- This is a partial solution/fix/something of the issue https://github.com/wagtail/wagtail/issues/6107
- The reason we want this attribute is (link to comment or a small summary) - this is helpful as the icons ticket is huge and it's unclear where to look on that issue for this specific fix.

Also, it would be good to have the word icons in the title, it's unclear at first glance on a list that this is about svg icons instead of svg image support elsewhere.

File: client/scss/core.scss [hunk 1]
```diff
@@ -198,4 +198,4 @@ These are classes that provide overrides.
 @import 'layouts/chooser-duplicate-upload';
 @import 'layouts/focal-point-chooser';
 @import 'layouts/modeladmin';
-@import 'layouts/redirects';
+@import 'layouts/redirects';
```
Comment (lb-) [inline client/scss/core.scss:201]: Can this unrelated change be removed - it seems like your editor may be set to remove lines at the end of the file. 

Here's an article on how to fix this in VSCode. 
https://blog.boot.dev/clean-code/line-breaks-vs-code-lf-vs-crlf/

**Comment by lb-** [comment]
Please can you take a look at the failing CI issues. Thank you

File: client/scss/core.scss (hunk 1 above)
Comment (vicky1312) [reply client/scss/core.scss:201]: Okay, I will work on it.

**Comment by vicky1312** [comment]
@LB I will work on the things you have listed. Thank you for the correction.

**Comment by vicky1312** [comment]
> Please can you take a look at the failing CI issues. Thank you

@lb- The failed CI for backend came as a result of the code I inserted in the homes.py file and the one for frontend is because of the unrelated change. I might have to revert this commit.

**Comment by zerolab** [comment]
closing in favour of #9393
